src/Microsoft.Data.SqlClient/netcore/src/Common/src/Microsoft/Data/Common/DbConnectionOptions.Common.cs

```diff
@@ -87,6 +87,9 @@ private static class KEY
             internal const string Persist_Security_Info = "persist security info";
             internal const string User_ID = "user id";
             internal const string AttachDBFileName = "attachdbfilename";
+            internal const string ClientCertificate = "client certificate";
+            internal const string ClientKey = "client key";
```

Comment:
Is this client certificate private key password? If so, please give an appropriate name

Reply:
ClientKey is private key file 
while  ClientKeyPassword is password for such private key (or it can be used if ClientCertificate is pfx format which alredy contains private key than ClientKeyPassword is used to decode Clientcertificate)